src/testing/match.go

```diff
@@ -5,9 +5,131 @@
 package testing
```

Comment:
I think this is missing copyright here?

Reply:
It's the file verbatim from upstream, and the first four lines are
```
// Copyright 2015 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

```
so I think it's set on copyright.

Reply:
I blame github web UI. :)